src/LuaTransformer.ts

```diff
@@ -3544,6 +3543,90 @@ export class LuaTransformer {
         }
     }
 
+    public transformConsoleCallExpression(expression: ts.CallExpression): ExpressionVisitResult {
+        const method = expression.expression as ts.PropertyAccessExpression;
+        const methodName = method.name.escapedText;
+
+        switch (methodName) {
+            case "log":
+                if (expression.arguments.length > 0) {
+                    if (ts.isStringLiteral(expression.arguments[0])
```

Comment:
It's probably a good idea to refactor this to a private helper method under this function:
```ts
private isStringFormatTemplate(expression: ts.Expression): boolean {
    return ts.isStringLiteral(expression) && expression.text.includes("%");
}
```